build.gradle

```diff
@@ -1,32 +1,34 @@
 plugins {
-  id "com.microsoft.azure.azurefunctions" version "1.1.0"
+	id "com.microsoft.azure.azurefunctions" version "1.2.0"
```

Comment:
use space instead of tab?

Reply:
The difference was: 

The original code had 2 spaces and mine 4 spaces. We should use 4 spaces in my humble opinion.